packages/backend/src/services/executionContractTrustGraph/scopeValidator.ts

```
@@ -0,0 +1,403 @@
+
+        const ownershipResult = await Promise.race([
+            input.ownershipValidator.validateOwnership(input.scopeTuple, {
+                abortSignal: abortController.signal,
+            }),
+            timeoutPromise,
+        ]);
+
+        return ownershipResult;
+    } catch (error: unknown) {
+        if (timeoutTriggered) {
+            throw new Error(OWNERSHIP_VALIDATOR_TIMEOUT_ERROR);
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)